index.js

```diff
@@ -17,7 +17,12 @@ var WinstonCloudWatch = function(options) {
   var awsSecretKey = options.awsSecretKey;
   var awsRegion = options.awsRegion;
   var messageFormatter = options.messageFormatter ? options.messageFormatter : function(log) {
-    return [ log.level, log.msg, stringify(log.meta) ].join(' - ');
+    if(stringify(log.meta)==="{}")
```

Comment:
No seems good now, I will do the changes and fix the tests later on this week, unless you're willing to have a go there.